src/check_jsonschema/cli.py

```diff
@@ -300,7 +308,7 @@ def build_instance_loader(args: ParseResult) -> InstanceLoader:
 
 def build_reporter(args: ParseResult) -> Reporter:
     cls = REPORTER_BY_NAME[args.output_format]
-    return cls(verbosity=args.verbosity)
+    return cls(color=args.color, verbosity=args.verbosity)
```

Comment:
I can make that change. There is merit to maintaining separation between Click and your application, such that if Click ever changes how they handle color you'll be able to work around it, or you'll be able to support other output forms, or other CLI libraries, etc., but since there's no interest in any of that this simplification seems worthwhile.

Reply:
Implemented in 0979071.